### PR TITLE
Add governance

### PR DESCRIPTION
**Pull request by RichiH**
Signed-off-by: Richard Hartmann <richih@richih.org>

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+The OpenObservability developers and community are expected to follow the values defined in the [Code of Conduct][coc]. Furthermore, the OpenObservability community strives for kindness, giving feedback effectively, and building a welcoming environment. The OpenObservability developers generally decide by consensus and only resort to conflict resolution by a majority vote if consensus cannot be reached.
+
+## Projects
+
+Each project must have a [`MAINTAINERS.md`][maintainers] file with at least one maintainer. Where a project has a release process, access and documentation should be such that more than one person can perform a release. Releases should be announced on the [public mailing list][public]. Any new projects should be first proposed on the [team mailing list][team] following the voting procedures listed below.
+
+## Decision making
+
+### Project lead
```
Comment (SuperQ) [inline GOVERNANCE.md:26]: If we're going to go with a project lead, maybe look at the [coredns governance](https://github.com/coredns/coredns/blob/master/GOVERNANCE.md#project-lead) wording.

Comment (RichiH) [reply GOVERNANCE.md:26]: CoreDNS has 24 maintainers. OO11y has four maintainers. Getting to substantially more maintainers from a diverse set of backgrounds is highly unlikely.

As such, our governance and thus project is extremely brittle. While I am not a lawyer, I have written, negotiated, and dismantled more legal texts in my life than I can reasonably count. There's no way to maintain the spirit of the law with the letter of the law unless we have a strong and stable provision in the governance.

SABDFL was the plan from day one, though admittedly we should have codified our governance earlier. Other than in extremely rare cases, in particular exemplar support, no overrides have been pulled, and that matches my long-term intention, overall morality, and philosophy.

**Comment by brian-brazil** [comment]
:+1:

**Review by robskillington — APPROVED** [review]
LGTM

File: GOVERNANCE.md (hunk 1 above)
Comment (RichiH) [reply GOVERNANCE.md:26]: Got principal OK in chat, please also put into PR.

**Comment by SuperQ** [comment]
LGTM

**Pull request merged**